powerline_shell/__init__.py

```diff
@@ -199,7 +201,13 @@ def main():
         print(json.dumps(DEFAULT_CONFIG, indent=2))
         return 0
 
-    config_path = find_config()
+    if args.config_file and not os.path.exists(os.path.expanduser(args.config_file)):
+        print("Cannot find config file: {}".format(args.config_file), file=sys.stderr)
+        return 1
+    elif args.config_file:
+        config_path = args.config_file
+    else:
+        config_path = find_config()
```

Comment:
This code needs to be updated to use `args.config_file`

Reply:
I'm actually not quite sure what you mean. If `args.config_file` is supplied, it doesn't call `find_config`; instead it just uses what the user sent. (?)

Reply:
Hm - I must have been loopy when I wrote that comment. Because I meant to comment on the lines above it and say that `args.config_file` needs to be changed to `args.config`.